content/howto/user-management.mdx

```diff
@@ -95,7 +95,7 @@ To search a customer using UID, Email or phone number, enter the respective valu
 
 You can also search for the customers based on the selected date range; this date range applies to the customers’ registration date.
 
-## <Pro>Add Users</Pro>
```

Comment:
@nickc95 This tag should be here as we are only supporting add users for the premium plan.

Reply:
Got to know that Add Users is now part of the free plan.